firmware/Core/Src/telecommands/antenna_telecommand_defs.c

```diff
@@ -354,18 +354,24 @@ uint8_t TCMDEXEC_ant_report_antenna_deployment_activation_time(const char *args_
     return 0;
 }
 
-/// @brief Telecommand: Measure the temperature of the antenna controller
+/// @brief Telecommand: Measures the temperature of the antenna controller in centi-degrees celsius
 /// @param args_str No args
 /// @return 0 on success, >0 on error
 uint8_t TCMDEXEC_ant_measure_temp(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
                         char *response_output_buf, uint16_t response_output_buf_len) {
-    uint16_t result;
-    const uint8_t comms_err = ANT_CMD_measure_temp(&result);
+    uint16_t measurement;
+    const uint8_t comms_err = ANT_CMD_measure_temp(&measurement);
     if (comms_err != 0) {
         snprintf(response_output_buf, response_output_buf_len, "Error: %d", comms_err);
         return comms_err;
     }
-
-    snprintf(response_output_buf, response_output_buf_len, "Success, Temp measurement: %u", result);
+    //TODO: remove the newline in between the strings to get a JSON string
+    snprintf(
+        response_output_buf, 
+        response_output_buf_len, 
+        "{\"raw_measurement\":%d," "\n" "\"temp_measurement_centi_degrees_celsius\":%d}", 
```

Comment:
I love these names normally! We may be logging temperature quite a bit, so maybe we can make them a bit less verbose (e.g., `raw_temp` and `temp_cC` match what I've seen/written in other hardware interfaces for this project).